core/metamodel/src/main/java/org/apache/isis/core/metamodel/facets/value/datejdk8local/Jdk8LocalDateUtil.java

```diff
@@ -1,170 +0,0 @@
-/*
```

Comment:
What's happened to this file?  It doesn't seem to have moved, is just deleted,.   Ditto the other classes deleted in the same area.